blip-00xx.md

```diff
@@ -0,0 +1,220 @@
+be:
+
+    639 = 68+num_hops(79) 
+    num_hops = 7
+
+#### QR code limit
+
+Another soft maximum value to keep in mind is the maximum number of bytes that
+can fit into a [QR code][qr] which is 2,953 bytes. This is a soft maximum 
+because this only applies if the invoice is in fact being transmitted via QR 
+code. This limit does not apply if the invoice is being transmitted via other
+protocols such as LNURL. In the cases where the limit does apply, then two 
+variables will be at play: 
+
+- The number of blinded paths
+- The number of blinded hops within each path (which will always also be 
+  restricted by the `data_length` maximum).
+
+The exact limit on the number of blinded paths that can be included depends on
+the size of other fields in the invoice. It is worth noting that an invoice with
+a blinded path should not contain any `r` (route hint) fields.
+
+## Universality
+
+This proposal is a temporary measure that will allow users to start making use
+of blinded paths in the context of payments and thereby taking advantage of the 
+potential privacy and payment success rate benefits that in they will in theory 
+provide. Once the Offers protocol along with its new invoice format has been 
+widely deployed, then there will be no use for this BOLT 11 carve-out. Due to 
+the forested temporary use of the new field, it makes sense to be in bLIP form
+rather than adding this in a more temporary way to the spec via a BOLT update 
+proposal.
+
+## Backwards Compatibility
+
+BOLT 11 states that the reader of an invoice "MUST skip over unknown fields". 
+This means that an un-updated reader of an invoice that included the new tagged 
+field would skip it when parsing the invoice and would likely attempt to find a
+path to the blinded destination key which would result in payment failure. This
+is, however, not a backward in-compatible change as the reader would still be
+able to parse the invoice. 
+
+## Reference Implementations
+
+The proposed encoding of the new BOLT 11 tagged-field is added to the LND 
+implementation in [this PR][impl].
+
+[tagged-fields]: https://github.com/lightning/bolts/blob/master/11-payment-encoding.md#tagged-fields
+[bolt11]: https://github.com/lightning/bolts/blob/master/11-payment-encoding.md
+[blinded-paths]: https://github.com/lightning/bolts/blob/master/04-onion-routing.md#route-blinding
+[offers]: https://github.com/lightning/bolts/pull/798
+[impl]: https://github.com/lightningnetwork/lnd/pull/8752
+[qr]: https://en.wikipedia.org/wiki/QR_code#Information_capacity
+[lnurl]: https://github.com/lnurl/luds
+
+
+# Some Questions: 
+
+- Does it make sense to add an experimental tagged field range for BOLT 11 in 
+  the bLIP2 spec or would this be over-kill since BOLT 11 is in any-case being 
+  phased out?
+- Would a new experimental feature bit (in the `I` context) make sense here so 
+  that the reader can fail fast instead of attempting to find a node that does 
```

Comment:
Should we include some test-vectors for the invoice with a blinded path, I think we have them ready in the LND PR I think ?

Reply:
ah indeed - good idea. Will add